app/src/main/java/be/brunoparmentier/openbikesharing/app/activities/MapActivity.java

```diff
@@ -96,6 +99,16 @@ protected void onCreate(Bundle savedInstanceState) {
         stationsDataSource = new StationsDataSource(this);
         ArrayList<Station> stations = stationsDataSource.getStations();
 
+        final Context context = getApplicationContext();
+        Configuration.getInstance().load(context, PreferenceManager.getDefaultSharedPreferences(context));
+        int versionCode = 0;
+        try {
+            versionCode = context.getPackageManager().getPackageInfo(getPackageName(), 0).versionCode;
+        } catch (PackageManager.NameNotFoundException e) {
+            e.printStackTrace();
+        }
+        Configuration.getInstance().setUserAgentValue(context.getPackageName() + "/" + versionCode);
```

Comment:
I don't think we really need the versionCode in the UserAgent + I'd use `BuildConfig.APPLICATION_ID` which identifies the app, instead of `context.getPackageName()` (the package name can be different from the application ID).

So I'd replace the whole block with:
```
Configuration.getInstance().setUserAgentValue(BuildConfig.APPLICATION_ID)
```
(as suggested in https://github.com/osmdroid/osmdroid/wiki/Important-notes-on-using-osmdroid-in-your-app#set-the-http-user-agent-variable)